pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -129,6 +129,6 @@ filterExisting() {
 wrapProgram() {
     local prog="$1"
     local hidden="$(dirname "$prog")/.$(basename "$prog")"-wrapped
-    mv $prog $hidden
-    makeWrapper $hidden $prog --argv0 '"$0"' "$@"
+    mv "$prog" "$hidden"
+    makeWrapper "$hidden" "$prog" --argv0 '$0' "$@"
```

Comment:
`$argv0` is already quoted now; see the line above (which you've also commented).